src/pages/LegalPages/styles.ts

```diff
@@ -0,0 +1,129 @@
+import { theme } from '@theme';
+import styled from 'styled-components';
+
+export const LegalPageWrapper = styled.main`
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  justify-content: space-between;
+  background: linear-gradient(
+    180deg,
+    ${theme.palette.background.default} 0%,
+    ${theme.palette.loginColors?.pageWrapperBg || '#f9fafb'} 100%
+  );
+`;
+
+export const LegalContent = styled.section`
+  flex: 1;
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  padding: 50px 80px;
+  color: ${theme.palette.text.primary};
+  line-height: 1.6;
+
+  hr {
+    display: none;
+    margin: 0;
+    padding: 0;
+    border: none;
+    height: 0;
+  }
+
+  & > div {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 100%;
+    max-width: 1062px;
+    padding: 32px 64px 64px 64px;
+    background: ${theme.palette.loginColors?.loginBg || theme.palette.background.paper};
+    border-radius: 20px;
+    border: 1px solid ${theme.palette.loginColors?.border ?? theme.palette.divider};
+    box-shadow: 0 2px 8px rgba(15, 23, 42, 0.04);
+    gap: 24px;
+  }
+
+  & div.markdown {
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+    width: 100%;
+    gap: 16px;
+  }
+
+  & div.markdown p,
+  li {
+    color: color: ${theme.palette.loginColors?.legalText};
+    font-family: ${theme.typography.fontFamily || "'Inter', sans-serif"};
+    font-size: ${theme.typography.body1?.fontSize || '16px'};
+    font-style: normal;
+    font-weight: ${theme.typography.body1?.fontWeight || 400};
+    line-height: ${theme.typography.body1?.lineHeight || '24px'};
+    max-width: 100%;
+    text-align: justify;
+    margin: 0 0 10px;
+  }
+
+  & div.markdown h1 {
+    color: ${theme.palette.loginColors?.titleColor};
+    text-align: center;
+    font-family: ${theme.typography.h5?.fontFamily || "'Poppins', sans-serif"};
+    font-size: ${theme.typography.h5?.fontSize || '24px'};
+    font-style: normal;
+    font-weight: ${theme.typography.h5?.fontWeight || 400};
```

Comment:
no hardcodded value, why theme.typography.h5?.fontWeight make it theme.typography.h5
.fontWeight